docs/src/content/docs/getting-started/learning.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Learning
+description: Explaining basic concepts, primitives and tools 
+sidebar: 
+  order: 2
+---
+## Installation
+
+First of all we will use @reatom/framework that incorparates many functions you may need in development.
+
+```bash
+npm install --save @reatom/framework
+```
+
+## Atoms are storing data and dependencies
+
+Let's have trivial example that employs reatom's reactivity by implementing simple `a + b = c`:
+
+```typescript
+import { createCtx, atom } from "@reatom/framework"
+
+const ctx = createCtx()
+
+const aAtom = atom(1, 'aAtom')
+const bAtom = atom(2, 'bAtom')
+
+const cAtom = atom((ctx) =>
+  ctx.spy(aAtom) + ctx.spy(bAtom),
+  'cAtom'
+)
+
+ctx.subscribe(cAtom, (c) => {
+  const a = ctx.get(aAtom);
+  const b = ctx.get(bAtom);
+  console.log(`${a} + ${b} = ${c}`)
+})
```

Comment:
Computation in the `subscribe` function is what you should avoid initially. It should serve as a bridge for side effects only.